demo/build.gradle

```diff
@@ -45,4 +45,5 @@ dependencies {
     withExtensionsCompile project(path: ':extension-flac')
     withExtensionsCompile project(path: ':extension-opus')
     withExtensionsCompile project(path: ':extension-vp9')
+    compile 'net.butterflytv.utils:rtmp-client:0.2.6.1'
```

Comment:
We don't want this dependency from the demo app. For all other cases where we've taken an external dependency, we've done so by creating an extension (see the six existing extensions [here](https://github.com/google/ExoPlayer/tree/release-v2/extensions)).

If you restructure this change so that `RtmpDataSource` is in a proper extension then we'll consider accepting a pull request. We probably don't want any samples in the demo app by default.